packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -335,7 +340,7 @@ const Chart = props => {
     const max_ticks = granularity === 0 ? 8 : 24;
 
     if (!symbol || active_symbols.length === 0) return null;
-
+    const activeSymbols = JSON.parse(JSON.stringify(active_symbols));
```

Comment:
This is gonna create new `activeSymbols` every time. Why do we need to create it on every render?

Reply:
@balakrishna-binary  oops, you are right, just move it to the properties